abis_mapping/utils/iri_patterns.py

```diff
@@ -0,0 +1,139 @@
+"""This module contains functions for constructing IRIs according to common patterns.
+
+This is important when the exact same IRI needs to be constructed from multiple template
+mappings so that the output RDF links together on these IRIs."""
+
+# third party
+import rdflib
+
+# local
+from abis_mapping import utils
+
+# typing
+from typing import Literal
+
+
+def survey_iri(
+    base_iri: rdflib.Namespace | None,
+    *,
```

Comment:
Note sure exactly what you are asking...

I made the other functions take all positional args because they are functions that will get called in a lot of places.
Making their args positional-only, makes all their calls consistent (there won't be some calls using kwargs), and compact (more likely to fit on one line).

Reply:
Similarly I made `survey_id` arg here keyword-only so all the callers are consistent. I wasn't worried about compactness since it is only called in a few places.

Reply:
I'm asking why survey_id is keyword only and yet every other function in  this module is positional only. Seems inconsistent.

Reply:
ok, I'll make it consistent with the others.